Switch to integer based streaming aggregation output batch session (#25029)
prestodb/presto: presto-docs/src/main/sphinx/presto_cpp/properties-session.rst

A higher value allows more memory to be allocated for scaling up table scans,
while a lower value limits the amount of memory used.

``native_streaming_aggregation_min_output_batch_rows``
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

* **Type:** ``integer``
* **Default value:** ``0``

> In streaming aggregation, wait until we have enough number of output rows

Reviewer: ```suggestion
In streaming aggregation, wait until there are enough output rows 
```

> to produce a batch of size specified by this. If set to 0, then

Reviewer: ```suggestion
to produce a batch of the size specified by this property. If set to ``0``, then 
```

> Operator::outputBatchRows will be used as the min output batch rows.

Reviewer: ```suggestion
``Operator::outputBatchRows`` is used as the minimum number of output batch rows.
```
